build.sbt

```diff
@@ -91,6 +90,7 @@ lazy val connector = (project in file("connector"))
       "aopalliance" % "aopalliance" % "1.0" % "provided",
       "org.codehaus.jackson" % "jackson-core-asl" % "1.9.13" % "provided",
       "org.codehaus.jackson" % "jackson-mapper-asl" % "1.9.13" % "provided",
+      "com.google.guava" % "guava" % "30.1.1-jre",
```

Comment:
Please return guava to commonTestDependencies, as it is used by the acceptance test in `published`

Reply:
done.